### PR TITLE
[Multimodal] Environment ver 2 with vulnerability fix

### PR DESCRIPTION
**Pull request by vivek-dani**
Create acft-multimodal-gpu environment version 2, with vulnerability fix for pyarrow

### CODE REVIEW

File: assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/requirements.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ azureml-acft-contrib-hf-nlp=={{latest-pypi-version}}
 azureml-acft-multimodal-components=={{latest-pypi-version}}
 # needed by multimodal
 mltable=={{latest-pypi-version}}
-datasets==2.12.0
+datasets==2.14.7
```
Comment (rjaincc) [inline assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/requirements.txt:8]: If we have it in DockerFile, we can remove it from requirements.txt for now. 
Introduce it later once the dependencies are resolved.

**Review by rjaincc — APPROVED** [review]

File: assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/Dockerfile [hunk 2]
```diff
@@ -6,4 +6,7 @@ RUN apt-get -y update
 
 # Install required packages from pypi
 COPY requirements.txt .
-RUN pip install -r requirements.txt --no-cache-dir
\ No newline at end of file
+RUN pip install -r requirements.txt --no-cache-dir
+
+# Vulnerability fix
+RUN pip install datasets==2.14.7
```
Comment (PhaniShekhar) [inline assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/Dockerfile:12]: Why do we need to install it here, if the correct version is added in requirements.txt

**Review by PhaniShekhar — APPROVED** [review]

File: assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/Dockerfile (hunk 2 above)
Comment (vivek-dani) [reply assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/Dockerfile:12]: It'll get overwritten by azureml-metrics package. azureml-metrics uses lower version of datasets which has vulnerability

Comment (PhaniShekhar) [reply assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/Dockerfile:12]: Thanks for the explanation. Should we add a comment in the Dockerfile about this ?

**Review by PhaniShekhar — APPROVED** [review]

**Pull request merged**